pkg/utils/mount/mount.go

```diff
@@ -65,11 +85,17 @@ func IsMountpoint(path string) (bool, error) {
 
 	stat, err := os.Stat(realPath)
 	if err != nil {
+		if strings.Contains(err.Error(), "transport endpoint is not connected") {
+			return true, errdefs.ErrNotConnected
+		}
 		return false, err
 	}
 
 	parentStat, err := os.Stat(filepath.Dir(realPath))
 	if err != nil {
+		if strings.Contains(err.Error(), "transport endpoint is not connected") {
```

Comment:
This is not a reliable way to judge if the dir is a mount point. Moreover,  did you try to catch the standard Linux error code rather than matching error string?

Reply:
Yes, got it.